### PR TITLE
Setting sane networking defaults on LB ApplicationLoadBalancer and FargateEcsService

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: awsx/ecs/ec2Service.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ export class EC2Service extends schema.EC2Service {
                     args.loadBalancers ?? taskDefinition?.loadBalancers,
                 waitForSteadyState: !utils.ifUndefined(
                     args.continueBeforeSteadyState,
-                    false,
+                    true,
```
Comment (danielrbradley) [inline awsx/ecs/ec2Service.ts:82]: Actually, is this correct ... there's a `!` before the ifUndefined

If continue not defined, we use true, so wait is false.
I think this was actually already correct: if continue not defined, use false, so wait is true.

Not very clear either way 🤨

File: awsx/ecs/fargateService.ts [hunk 2]
```diff
@@ -79,7 +84,7 @@ export class FargateService extends schema.FargateService {
                     args.loadBalancers ?? taskDefinition?.loadBalancers,
                 waitForSteadyState: !utils.ifUndefined(
                     args.continueBeforeSteadyState,
-                    false,
+                    true,
```
Comment (danielrbradley) [inline awsx/ecs/fargateService.ts:87]: Same as above

File: awsx/ecs/ec2Service.ts (hunk 1 above)
Comment (stack72) [reply awsx/ecs/ec2Service.ts:82]: fixed!

File: awsx/ecs/fargateService.ts (hunk 2 above)
Comment (stack72) [reply awsx/ecs/fargateService.ts:87]: fixed!

**Pull request merged**